_posts/2020-12-01-introducing-the-library.md

```diff
@@ -1,10 +1,50 @@
 ---
 layout: post
-title:  "Introducing the Library"
+title:  "Introducing the Library by StreamSounds"
 author: ogjunkyard
 categories: [ about us ]
 image: "assets/images/logo.png"
 featured: true
 ---
 
-Something, something, write something here.
+Hey folks, OGJunkyard here.
+
+I wanted to take a moment and give you the backstory behind the Library by StreamSounds. I also wanted to tell you why I thought it would help the Twitch community. I ALSO wanted to tell you why I'm doing this, because I think transparency, openness, and honesty are important. So, this will broken into three parts:
+
+- The Backstory behind the Library by StreamSounds
+- Why this will be helpful to streamers
+- OGJunkyard's motivations and his promise to you
+
+### The Backstory behind the Library by StreamSounds
+
+I had been streaming on Twitch for two years solidly when I got the idea to post helpful content about what I learned about streaming without any strings attached. So, I started looking around the communities I was a part of, the people I was following on various websites, and looking at their content while thinking to myself, "How useful is this actually? Can I actually reproduce these results myself?" I had also started a company and thought this could be a good way to put value out into the world to start getting some traction and eyeballs on what the company offered, while also setting the company up for success when the product that I wanted to launch came out of alpha/beta.
+
+I started talking to one of my regular viewers, FlavCreations, who had been very helpful in the past. He ended up liking the idea and had been very helpful in another community we are a part of called Infinite Quality, so we decided to try to come up with a way to share our ideas. I pitched Flav the idea of putting out content that could be proven and repeatable, which he seemed to like, so we decided to give it a shot.
+
```

Comment:
Maybe add in something about why library was chosen over school/university/professional category names? Could be a cool add in.